<?xml version="1.0"?>

<launch>

    <arg name="dt" default="0.1"/>
    <arg name="map_width" default="10"/>
    <arg name="resolution" default="0.1"/>
    <arg name="search_range" default="30"/>
    <arg name="cost_collision" default="90"/>
    <arg name="min_cost" default="30"/>

    <group ns="local_costmap_maker">

        <node pkg="dynamic_obstacle_avoidance_planner" name="local_costmap" type="local_costmap" output="screen">

        </node>

        <node pkg="dynamic_obstacle_avoidance_planner" name="dynamic_local_costmap_maker" type="dynamic_local_costmap_maker" output="screen">
            <param name="DT" value="$(arg dt)"/>
            <param name="MAP_WIDTH" value="$(arg map_width)"/>
            <param name="RESOLUTION" value="$(arg resolution)"/>
            <param name="SEARCH_RANGE" value="$(arg search_range)"/>
            <param name="COST_COLLISION" value="$(arg cost_collision)"/>
            <param name="MIN_COST" value="$(arg min_cost)"/>
        </node>

    </group>

</launch>
